src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main__title {
+  font-size: 17px;
+  font-weight: 700;
+  margin: 0;
+}
+
+.main__subtitle {
+  font-size: 15px;
+  font-weight: 400;
+  margin: 1em 0;
+}
+
+.main__text {
+  margin: 0;
+}
+
+.main__link-extra-articles {
```

```suggestion
.main__link--extra {
```
try to give a shorter names for classes

---

Okay, I'll do it in the future!